src/App.jsx

```diff
@@ -14,70 +16,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
+  const isAdded = good => () => {
+    setSelectedGood(good);
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const isDeleted = () => {
```

Comment:
nit: isDeleted ->  resetGood

we usually name boolean variables starting with `is`